ale/drivers/base.py

```diff
@@ -165,8 +223,19 @@ def starting_ephemeris_time(self):
         return self._starting_ephemeris_time
 
     @property
-    def _exposure_duration(self):
-        return self._label['EXPOSURE_DURATION'].value * 0.001  # Scale to seconds
+    def exposure_duration(self):
+        try:
+            return self._label['EXPOSURE_DURATION'].value * 0.001  # Scale to seconds
+        except:
+            return np.nan
```

Comment:
Stick a warning in here so that we do not silently return a NaN?

Reply:
We need to decide on how to handle these kinds of failures. I think a warning and returning np.NaN works